UNRELEASED.md

```diff
@@ -1,2 +1,9 @@
 # Unreleased
 
+**New:**
+ - bpk-component-scrollable-calendar:
+   - implemented BpkScrollableCalendarDate, built on top of BpkCalendarDate
+
+**Changed:** 
+ - bpk-component-calendar:
+   - exposed propTypes as BpkCalendarPropTypes for reuse in BpkScrollableCalendarDate
```

Comment:
This should say: "Exposed `BpkCalendarDatePropTypes` for reuse in other components."

^ pending a name change of course :P